changelog.d/3630.feature

```diff
@@ -0,0 +1 @@
+Blocks registration and sign in if max mau value exceeded
```

Comment:
Sorry, can we quickly change this to "Add ability to limit number of monthly active users on the server", or some such. Otherwise its going to be confusing for people to read the changelog who haven't had the full context.